packages/example-getting-started/docs/juggler.md

```diff
@@ -1,5 +1,15 @@
 ### Adding the Legacy Juggler
 
+The Legacy Juggler is a "bridge" between the existing
+[loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler)
+and the new LoopBack 4 architecture. It provides the capabilities required to
+access datasources, and perform
```

Comment:
Include a one line of what a datasource is ... or better yet, remove that word and use something like, provides a persistence (storage) layer for your data.